src/sidebar/helpers/group-list-item-common.ts

```diff
@@ -1,5 +1,5 @@
 import type { Group } from '../../types/api';
 
 export function orgName(group: Group): string {
-  return group.organization && group.organization.name;
+  return group.organization.name;
```

Comment:
Our types define `organization` as mandatory, so this conditional logic is not needed. It's probably a leftover from before TypeScript was being used.